packages/asl-components/src/licence-status-banner/index.jsx

```diff
@@ -2,8 +2,9 @@ import React, { useState, useEffect } from 'react';
 import Snippet from '../snippet';
 import classnames from 'classnames';
 import { format } from 'date-fns';
+import { DATE_FORMAT } from '@asl/projects/client/constants';
```

Comment:
asl-components shouldn't be depending on asl-projects or it creates a circular dependency. There is a constant we can use here: https://github.com/UKHomeOffice/aspel-workspace/blob/main/packages/asl-components/src/utils.js#L40-L43. Eventually we should have a consistent format in one place (either here or in asl-constants) and projects should be updated so it's not formatting dates separately. That is out of scope for this ticket, but we shouldn't introduce this dependency where it's not needed.